CHANGELOG.md

```diff
@@ -37,14 +46,14 @@ include updated documentation and files outlined by the
 
 Repository structure changes include:
 
-* Migrating `pymods` directory to `swath_projector`.
-* Migrating `swotrepr.py` to `swath_projector/adapter.py`.
-* Addition of `swath_projector/main.py`.
+- Migrating `pymods` directory to `swath_projector`.
+- Migrating `swotrepr.py` to `swath_projector/adapter.py`.
+- Addition of `swath_projector/main.py`.
 
 For more information on internal releases prior to NASA open-source approval,
 see legacy-CHANGELOG.md.
 
-[v1.1.1]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.1.0)
-[v1.1.0]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.1)
-[v1.0.1]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.1)
-[v1.0.0]:(https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.0)
+[v1.1.1]: (https://github.com/nasa/harmony-swath-projector/releases/tag/1.1.0)
+[v1.1.0]: (https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.1)
+[v1.0.1]: (https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.1)
+[v1.0.0]: (https://github.com/nasa/harmony-swath-projector/releases/tag/1.0.0)
```

Comment:
Also, I don't this this is doing what we want it to be doing. The goal is to be able to click each version number to display the differences to the previous version.  Take a look at the [HyBIG CHANGELOG.md](https://raw.githubusercontent.com/nasa/harmony-browse-image-generator/refs/heads/main/CHANGELOG.md) to see how this is implemented.